pandas/core/arrays/base.py

```diff
@@ -32,6 +32,18 @@
 _extension_array_shared_docs = dict()  # type: Dict[str, str]
 
 
+def safe_ea_cast(cls, obj, dtype=None):
+    """
+    Call to cls._from_sequence that returns the object unchanged on Exception.
```

Comment:
Based on the way this function is used, the parameter name `cls` confuses me because I see `self` being passed in for some of the invocations.

This function is internal because an enumeration of the parameters couldn't hurt.

Reply:
> an enumeration of the parameters couldn't hurt.

Good idea.  I'll add those to the docstring